### ISSUE TITLE
sendMail has `any` result type

### ISSUE THREAD

**Issue by bruceharrison1984** [sofn-xyz/mailing#164]
The result of the `sendMail` call is currently typed as `any`. It'd be helpful if it was strongly typed.

```ts
  const ownerEmailResult = await sendMail({
    subject: 'Here's an email!',
    to: owner.body?.email,
    component: (
      <EventCreatedOwnerTemplate {...req.body} name={owner.body?.full_name!} />
    ),
    forceDeliver: true,
  });

  // ownerEmailResult is type <any>
```

**Comment by psugihara** [comment]
yeah, i like this idea. thanks for the issue!

**Comment by rashidmya** [comment]
can I be assigned this?

**Comment by psugihara** [comment]
go for it!

**Comment by iDuuck** [comment]
This is already implemented, isn't it? Look here: https://github.com/sofn-xyz/mailing/blob/main/packages/core/src/index.ts#L10-L14

**Comment by alexfarrill** [comment]
I think @bruceharrison1984 is saying that the return value should be typed.  If I'm not mistaken, that line you reference is showing that the parameters are typed

**Comment by psugihara** [comment]
hey @rashidmya, any luck here?

**Comment by psugihara** [comment]
Think we stumbled on a simple fix...
https://github.com/sofn-xyz/mailing/pull/278

**Comment by psugihara** [comment]
fixed in 0.8.8

thanks all!
